packages/gatsby/src/commands/build-html.ts

```diff
@@ -132,7 +120,7 @@ const doBuildPages = async (
   }
 }
 
-const buildPages = async ({
+const buildHTML = async ({
```

Comment:
Rather than a default export, how about a named export here? i.e. `export const buildPages = ` I generally prefer named over default exports because it helps catch typos. Do you agree?

Reply:
I've been back and forth on this. I do like named exports for that reason. The reason i've used export default is mostly so future devs think twice about exporting other methods. Not speaking about Gatsby devs per se, but devs in general tend to not think about API surface and are happy to export anything that "gets a job done."

I realize export default doesn't really solve this. But might lend in a small way to prevent api surface creep

Reply:
I agree that we want to avoid API creep, but there's no specific reason to only export one per file. It would make sense to agree a style guide for this sort of thing so we can be consistent

Reply:
I read into this more and i'm not pretty convinced the default export was a poor design for es modules.. happy to switch this to named exports